src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,41 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    public static final int MAX_SIZE = 12;
+    public static final int TOT = -1;
+    public static final int EMPTY = -1;
+    final private int[] numbers = new int[MAX_SIZE];
+
+    public int getTotal() {
+        return total;
+    }
+
+    private int total = EMPTY;
```

Comment:
This constant should be different (separate) than the one in peekaboo and countOut methods.